### DISCUSSION TITLE
yarn install --offline : unsupported option name

### DISCUSSION THREAD

**Discussion by AndrisJefimovs** [carbon-design-system/carbon#7845] [Help] [answered]
Hello everyone!

I was following the contribution guide, but I couldn't get [step 5](https://github.com/carbon-design-system/carbon/blob/master/.github/CONTRIBUTING.md#5-build-and-start-the-development-server) to work.

If I run `yarn install --offline`, I am getting the error `Unknown Syntax Error: Unsupported option name ("--offline").`.

As suggested in #5132 I installed Python 2 (2.7.15), but nothing changed.

**Comment by tw15egan** [comment, marked as answer]
Looks like that's a bit outdated, you should just need to run `yarn` at the root of the directory, then `yarn build`

**Reply by AndrisJefimovs** [reply]
Thank you! That was the issue. 👏

**Reply by tw15egan** [reply]
I've updated the documentation as well to reduce any future issues, thanks for pointing that out!
